opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -118,6 +118,17 @@ def __init__(self,
         self._client = client
         self._transport = transport(self)
         self._default_labels = default_labels
+        if self._options.resource == "":
+            monitored_resource = MonitoredResourceUtil.get_instance()
+            if monitored_resource is not None:
+                self._resource_type = monitored_resource.resource_type
+                self._resource_labels = monitored_resource.get_resource_labels()
+            else:
+                self._resource_type = GLOBAL_RESOURCE_TYPE
+                self._resource_labels = ""
```

Comment:
Other resource labels are dicts or none, should this be a string?

Reply:
Done.